easybuild/easyconfigs/m/MITObim/MITObim-1.9.1-gompi-2019b.eb

```
@@ -0,0 +1,33 @@
+easyblock = "CmdCp"
+
+name = 'MITObim'
+version = '1.9.1'
+
+homepage = "https://github.com/chrishah/MITObim"
+description = """The MITObim procedure (mitochondrial baiting and iterative mapping) represents
+ a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms
+ directly from total genomic DNA derived NGS reads."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+
+source_urls = ['https://raw.githubusercontent.com/chrishah/MITObim/v1.9.1/']
+sources = [{'filename': 'MITObim.pl', 'extract_cmd': "cp %s ."}]
+checksums = ['8f83be5391db8a282e8c193b23e53c8d99c9894fb6990ea06de1321207ba6dd3']
```

Same changes apply as in `foss/2018b`
```suggestion
github_account = 'chrishah'
source_urls = [GITHUB_SOURCE]
sources = ['v%(version)s.tar.gz']
checksums = ['8f7d47eb6ea677ca152492a477ee6c7542c0f853e921675b81be509ddd4f75bf']
```